static/css/siglens.css

```diff
@@ -2084,6 +2110,112 @@ div.dts div.dataTables_scrollBody table {
 #available-fields::-webkit-scrollbar {
     width: 0;
 }
+.metrics-dropdown {
+    display: none;
+    color: var(--navbar-link-text) !important;
+    text-decoration: none;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    left: 154px;
+    border-radius: 5px;
+    box-shadow: 0px 2px 8px 0px var(--border-shadow);
+    border: 1px solid var(--border-btn-color);
+}
+.metrics-dropdown a {
+    color: var(--navbar-link-text) !important;
+    text-decoration: none;
+} 
+.metrics-dropdown li::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    width: 2px;
+    background-color: transparent; 
+}
+.border-danger {
+    border: 2px solid red !important; /* Adjust border style as needed */
+    background-color: rgb(252, 234, 234) !important;
+}
+.addrulepopupOverlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5); /* semi-transparent black background */
+    display: none;
+    align-items: center;
+    justify-content: center;
+    z-index: 1000;
+  }
+  
+  .addrulepopupContent {
+    background: #fff;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+    text-align: center;
+    width: 300px;
+  }
+  
+  .header {
+    margin-top: 0;
+    margin-bottom: 20px;
+    font-size: 1.5em;
+  }
+  
+  label {
```

Comment:
here , you have given CSS style without and specific id or class ( label, input, button) 
Remove them and it should fix the other screens.